leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -537,10 +544,33 @@ private void changeLocationInfoLayoutVisibility(int visibility) {
     locationInfoLayout.setVisibility(visibility);
   }
 
+  private void showCoordinatesLayout() {
+    this.longitude.setVisibility(View.VISIBLE);
+    this.latitude.setVisibility(View.VISIBLE);
+    this.coordinates.setVisibility(View.VISIBLE);
+    this.street.setVisibility(View.GONE);
+    this.city.setVisibility(View.GONE);
+    this.zipCode.setVisibility(View.GONE);
```

Comment:
Te word "this" in this case, is unnecessary to reference the activity attributes